graph/schema.graphql

```diff
@@ -0,0 +1,52 @@
+type Document {
+  id: ID!
+  title: String!
+  key: String!
+  kind: String!
+  project: Project!
+}
+
+enum DocumentType {
+  VPAT
+  DesignSessionDeck
+  TRBLetter
+}
+
+input DocumentInput {
+  title: String!
+  key: String!
+  type: DocumentType!
+  project_id: ID!
+}
+
+type DocumentUpdateResponse {
+  success: Boolean!
+  message: String
+  document: Document!
+}
+
+type Project {
+  id: ID!
+  name: String!
+  documents: [Document!]!
+}
+
+input ProjectInput {
+  name: String!
+}
+
+type ProjectUpdateResponse {
+  success: Boolean!
+  message: String
+  project: Project!
+}
+
+type Query {
+  projects: [Project!]!
```

Comment:
Possibly a nit this early in the process, but thinking about pagination and/or filtering now might be a good idea.

The best practices wiki has a couple of links that influenced me, but one in particular was the [Building Evolvable Schemas](https://www.apollographql.com/blog/graphql-schema-design-building-evolvable-schemas-1501f3c59ed5/). It suggests maybe making a `Projects` object so you can add additional fields.  That might also help with pagination.